image.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <link rel="stylesheet" href="image.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <header>
+        <h1>CSS.SUP</h1>
+
+        <div class="con">
+            <button ><a href="https://github.com/bipin7yadav" class="fa fa-github icon"></a></button>
+            <button ><a href="https://twitter.com/bipinyadav9769" class="fa fa-twitter icon"></a></button>
+            <button ><a href="https://www.linkedin.com/in/bipin-yadav-07a08217a/" class="fa fa-linkedin icon"></a></button>
+
+        </div>
+        
+    </header>
+    <div class="container">
+
+
+        <div class="sidebar">
+            <h3><a href="/component library landing page/index.html" class="side-nav">Home</a></h3>
+            <h3><a href="/avatar comp/avatar.html" class="side-nav">Avatar</a></h3>
+            <h3><a href="/alert component/alert.html" class="side-nav">Alert</a></h3>
+            <h3><a href="/badge component/badge.html" class="side-nav">Badge</a></h3>
+            <h3><a href="/Button componet/button.html" class="side-nav">Button</a></h3>
+            <h3><a href="/card component/card.html" class="side-nav">Card</a></h3>
+            <h3><a href="/img  comp/image.html" class="side-nav">Image</a></h3>
+            <h3><a href="/Input comp/input.html" class="side-nav">Input</a></h3>
+            <h3><a href="/list Component/list.html" class="side-nav">List</a></h3>
+            <h3><a href="/Navigation comp/navigation.html" class="side-nav">Navigation</a></h3>
+            <h3><a href="/rating comp/rating.html" class="side-nav">Rating</a></h3>
+            <h3><a href="/slider comp/slider.html" class="side-nav">Slider</a></h3>
+            <h3><a href="/text utilities/text-utility.html" class="side-nav">Text utilities</a></h3>
+      
```

Comment:
These gaps between elements don’t look good. Try clearing it all before pushing changes.

Reply:
sure